app/components-react/root/NewsBanner.tsx

```diff
@@ -0,0 +1,83 @@
+import { shell } from 'electron';
+import { $t } from '../../services/i18n';
+import React, { useState } from 'react';
+import { Services } from '../service-provider';
+import cx from 'classnames';
+import styles from './NewsBanner.m.less';
+import { TAppPage } from '../../services/navigation';
+import { useVuex } from '../hooks';
+
+export default () => {
+  const { AnnouncementsService, NavigationService, WindowsService, SettingsService } = Services;
+
+  const [processingClose, setProcessingClose] = useState(false);
+
+  const v = useVuex(() => ({
+    currentBanner: AnnouncementsService.state,
+    bannerExists: AnnouncementsService.bannerExists,
+  }));
+
+  // TODO: use hook after Alex's branch is merged
+  // const toggleAnimation = () => {
+  //   WindowsService.actions.updateStyleBlockers('main', true);
+  //   window.setTimeout(() => {
+  //     WindowsService.actions.updateStyleBlockers('main', false);
+  //   }, 500);
+  // };
+
+  const closeBanner = async (
```

Comment:
Use simple named functions for the component methods. We don't need arrow functions here because we don't use `this` anymore. Also, named functions allow you to define them in any order